stories/chip.stories.js

```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react'
+
+import { Chip } from '../src'
+import { Add, Close } from '../src/icons'
+
+
+export default {
+  title: 'Chip',
+  component: Chip,
+}
+
+const description = {
+  p: 'text-black mt-5 mb-1',
+}
+
+const sampleData = [
```

Comment:
I suggest using our color for the sample. If the Zeplin access is blocking you to get the inspection. Let us know.

Reply:
@kc-leung If you need any help on getting inspect from Zeplin, let us know.